src/amp.js

```diff
@@ -63,19 +63,17 @@ try {
 
       maybeValidate(window);
       makeBodyVisible(document, waitForExtensions(window));
-    } catch (e) {
-      makeBodyVisible(document);
     } finally {
+      makeBodyVisible(document);
```

Comment:
This will immediately make the body visible instead of waiting for the Dynamic CSS extension to load.